From bac2f3ee35f1d13be0e90301365c93accb27a3f4 Mon Sep 17 00:00:00 2001
From: PatrykKups <pkups@anaconda.com>
Date: Tue, 24 Jun 2025 20:37:46 +0200
Subject: [PATCH] patch

---
 pyproject.toml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 45d3da8..f8abaa3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -19,8 +19,7 @@ maintainers = [
     {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
     {name = "Thomas A Caswell", email = "tcaswell@bnl.gov"},
 ]
-license = "BSD-3-Clause"
-license-files = ["LICENSE", "lzf/LICENSE.txt", "licenses/*"]
+license = {text = "BSD-3-Clause"}
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-- 
2.39.5 (Apple Git-154)

