From c2608a2295e4b588bb6a8b9d7c5929a073323b61 Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Mon, 24 Nov 2025 10:54:08 +0100
Subject: [PATCH] patch

---
 pyproject.toml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 604efb9..3d156ca 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,8 +22,7 @@ maintainers = [
     {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
     {name = "Thomas A Caswell", email = "tcaswell@bnl.gov"},
 ]
-license = "BSD-3-Clause"
-license-files = ["LICENSE", "lzf/LICENSE.txt", "licenses/*"]
+license = {text = "BSD-3-Clause"}
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-- 
2.45.2

