From 5579f5632760bf852988c08e89c650dcdfed9ad3 Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Tue, 4 Nov 2025 13:51:23 +0100
Subject: [PATCH] patch

---
 pyproject.toml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 604efb9..3d156ca 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,8 +22,7 @@ maintainers = [
     {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
     {name = "Thomas A Caswell", email = "tcaswell@bnl.gov"},
 ]
-license = "BSD-3-Clause"
-license-files = ["LICENSE", "lzf/LICENSE.txt", "licenses/*"]
+license = {text = "BSD-3-Clause"}
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-- 
2.39.5 (Apple Git-154)

